### DISCUSSION TITLE
Motion Blur In Godot

### DISCUSSION THREAD

**Discussion by NeonfireStudio** [godotengine/godot-proposals#9665] [3D]
### Describe the project you are working on

A Horror Game

### Describe the problem or limitation you are having in your project

I want to add motion blur system

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add a motion blur system in camera attributes, there will be options for enabling motion blur on walking or rotating camera

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

...

### If this enhancement will not be used often, can it be worked around with a few lines of script?

...

### Is there a reason why this should be core and not an add-on in the asset library?

I think its not possible for creating for users

**Comment by LunaCapra** [comment]
This is already a proposal: #2933
